http/middleware.go

```diff
@@ -125,13 +125,14 @@ func ignoreMethod(ignoredMethods ...string) isValidMethodFn {
 	}
 }
 
+// TODO(@jsteenb2): make this a stronger type that handlers can register routes that should not be logged.
 var blacklistEndpoints = map[string]isValidMethodFn{
 	prefixSignIn:                     ignoreMethod(),
 	prefixSignOut:                    ignoreMethod(),
 	prefixMe:                         ignoreMethod(),
 	mePasswordPath:                   ignoreMethod(),
 	usersPasswordPath:                ignoreMethod(),
-	prefixPackages + "/apply":        ignoreMethod(),
+	"/api/v2/packages/apply":         ignoreMethod(),
```

Comment:
not sure the reason behind this change

Reply:
right now its to decouple the routes in this manner. Ideally we'd have a means of registering these paths instead of hardcoding them. This change is necessary b/c `prefixPackages` is no long er in http.